package.json

```diff
@@ -20,7 +20,8 @@
   "devDependencies": {
     "component-as-module": "0.3.0",
     "eslint": "0.12.0",
-    "nodeunit": "0.10.2"
+    "nodeunit": "0.10.2",
+    "superagent": "^3.3.1"
```

Comment:
When running the tests it requires superagent. A peer dependency isn't enough to have it installed by npm install (only on old npm versions).
**Source:** http://blog.npmjs.org/post/110924823920/npm-weekly-5